### ISSUE TITLE
what does "1sent+1sur" mean?

### ISSUE THREAD

**Issue by huidemie** [nusnlp/contextemb-wsd#3]
hello, I don't understand "1sent+1sur" in the paper,and I didn't find it in the code. Can you tell me what is the "surrounding"?

**Comment by chrhad** [comment]
Hi, 1sent+1sur means that the input consists of the sentence that contains the ambiguous word to disambiguate. It is activated by adding the parameter `--num-context 1`. Please refer to the [example running script](https://github.com/nusnlp/contextemb-wsd/blob/master/runall_s1.bash#L10). The surrounding sentence refers to the sentence before and after the sentence that contains the word to disambiguate.

**Comment by huidemie** [comment]
> Hi, 1sent+1sur means that the input consists of the sentence that contains the ambiguous word to disambiguate. It is activated by adding the parameter `--num-context 1`. Please refer to the [example running script](https://github.com/nusnlp/contextemb-wsd/blob/master/runall_s1.bash#L10). The surrounding sentence refers to the sentence before and after the sentence that contains the word to disambiguate.

Thank you for your reply! Another question : you got the state-of-the-art F1 using BERT without finetune?

**Comment by chrhad** [comment]
> Thank you for your reply! Another question : you got the state-of-the-art F1 using BERT without finetune?

Yes, we did not fine-tune the BERT model and got the reported state-of-the-art results. However, the sense output layer parameters, which are randomly initialized, are updated during training.
